packages/prompts/src/index.ts

```diff
@@ -995,3 +1024,80 @@ export const tasks = async (tasks: Task[], opts?: CommonOptions) => {
 		s.stop(result || task.title);
 	}
 };
+
+export interface TaskLogOptions extends CommonOptions {
+	message: string;
```

Comment:
`message` feels a bit too close to the `taskLog.message(...)` method and makes it seem like they are connected. i think it might be clearer if it was just called `title`.
```suggestion
	title: string;
```

Reply:
this is on purpose because pretty much every other prompt uses `message` for the primary message

i understand what you're saying, but is that worth breaking consistency?

Reply:
it shouldn't be that inconsistent given that `note`'s and `task`'s also provide `title` props: https://github.com/bombshell-dev/clack/blob/3552578ad867f144e10cf33dbd4d46960cace0c9/packages/prompts/src/index.ts#L642

https://github.com/bombshell-dev/clack/blob/3552578ad867f144e10cf33dbd4d46960cace0c9/packages/prompts/src/index.ts#L973-L987